src/hooks/useAuth.tsx

```
@@ -0,0 +1,110 @@
+  }
+  return context;
+};
+
+export const AuthProvider: React.FC<{children: React.ReactNode}> = ({children}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const {t} = useTranslationSafe();
+
+  useEffect(() => {
+    loadUserFromStorage();
+  }, []);
```

ESLint

[error] React Hook useEffect has a missing dependency: 'loadUserFromStorage'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)